libs/auth/src/common/services/auth-request/auth-request.service.ts

```diff
@@ -116,7 +119,7 @@ export class AuthRequestService implements AuthRequestServiceAbstraction {
         Utils.fromUtf8ToArray(masterKeyHash),
         pubKey,
       );
-      keyToEncrypt = masterKey.encKey;
+      keyToEncrypt = (masterKey.inner() as Aes256CbcKey).encryptionKey;
```

Comment:
I don't think this cast is needed:
```suggestion
      keyToEncrypt = masterKey.inner().encryptionKey;
```

Reply:
Removed the cast